.github/workflows/merge-dependencies.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Merge Dependencies'
+
+# https://github.com/ahmadnassri/action-dependabot-auto-merge/issues/60#issuecomment-806027389
+on: [pull_request_target]
+
+jobs:
+  auto-merge:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [12.x]
```

Comment:
Matrix is not needed if it uses only one OS and one Node.js version

Reply:
But does it hurt having a matrix? It is more extensible.

Reply:
Well, we never use a matrix if we don't need it and I find it confusing when there is a matrix without using it.